### PR TITLE
Team skill

### PR DESCRIPTION
**Pull request by bratke**
```
postgres=# select trueskillmean,trueskillstandarddeviation,id from player;
  trueskillmean   | trueskillstandarddeviation |    id     
------------------+----------------------------+-----------
 29.1753579557308 |           5.95432253666433 | U12SQMB5W
 17.7505020734372 |           5.95432253666433 | UGZT9UB3L
 26.3018228313482 |           5.95432253666433 | U12JD3LQG
 26.7723171394838 |           5.95432253666433 | U12PAFQ9E
(4 rows)

postgres=# select * from team;
  trueskillmean   | trueskillstandarddeviation | player2_id | player1_id 
------------------+----------------------------+------------+------------
 20.7943586078794 |           7.19471815777177 | UGZT9UB3L  | U12JD3LQG
 29.2056413921206 |           7.19471815777177 | U12SQMB5W  | U12PAFQ9E
 20.7943586078794 |           7.19471815777177 | UGZT9UB3L  | U12PAFQ9E
 29.2056413921206 |           7.19471815777177 | U12SQMB5W  | U12JD3LQG
 23.0050378955886 |           5.31163635492655 | UGZT9UB3L  | U12SQMB5W
 26.9949621044114 |           5.31163635492655 | U12PAFQ9E  | U12JD3LQG
(6 rows)

postgres=# select count(*) from tournament;
 count 
-------
     5
(1 row)
```

### CODE REVIEW

File: src/main/java/eu/m6r/kicker/trueskill/TrueSkillCalculator.java [hunk 1]
```diff
@@ -91,8 +91,53 @@ public Tournament updateRatings(final Tournament tournament) {
...
+            return tournament;
+        }
+    }
+
     private double playerSkill(final eu.m6r.kicker.models.Player player) {
-        final var rating = new Rating(player.trueSkillMean, player.trueSkillStandardDeviation);
+        return getRating(player.trueSkillMean, player.trueSkillStandardDeviation);
+    }
+
+    private double teamSkill(eu.m6r.kicker.models.Team team) {
```
Comment (gesundkrank) [inline src/main/java/eu/m6r/kicker/trueskill/TrueSkillCalculator.java]: `final`

File: src/main/java/eu/m6r/kicker/trueskill/TrueSkillCalculator.java [hunk 2]
```diff
@@ -91,8 +91,53 @@ public Tournament updateRatings(final Tournament tournament) {
...
+        return getRating(team.trueSkillMean, team.trueSkillStandardDeviation);
+    }
+
+    private double getRating(Double trueSkillMean, Double trueSkillStandardDeviation) {
```
Comment (gesundkrank) [inline src/main/java/eu/m6r/kicker/trueskill/TrueSkillCalculator.java]: final

File: src/main/java/eu/m6r/kicker/trueskill/TrueSkillCalculator.java [hunk 3]
```diff
@@ -108,6 +153,13 @@ private ITeam toTeam(final IPlayer trueSkillPlayer1,
         return team;
     }
 
+    private ITeam toTeam(IPlayer trueSkillTeam, eu.m6r.kicker.models.Team team) {
```
Comment (gesundkrank) [inline src/main/java/eu/m6r/kicker/trueskill/TrueSkillCalculator.java]: final, final

**Review by gesundkrank — APPROVED** [review]
Three very minor comments. LGTM
Please only merge after everything is prepared for the deployment.

**Comment by bratke** [comment]
```
postgres=# select teamaplayer1skillchange, teamaplayer2skillchange, teamaskillchange, teambplayer1skillchange, teambplayer2skillchange, teambskillchange  from tournament order by date;
 teamaplayer1skillchange | teamaplayer2skillchange | teamaskillchange  | teambplayer1skillchange | teambplayer2skillchange |  teambskillchange  
-------------------------+-------------------------+-------------------+-------------------------+-------------------------+--------------------
        4.61855737022626 |        4.61855737022626 |  7.62148691880531 |       -1.32911772498836 |       -1.32911772498836 | -0.789795865435902
       -2.57225335878822 |       -2.57225335878822 | -2.38681064227509 |        6.14866404895366 |        6.14866404895366 |    9.0807148873002
        3.83933299626919 |        3.83933299626919 |  7.62148691880531 |       -1.28645273786195 |       -1.43253557348142 | -0.789795865435902
(3 rows)

postgres=#  select trueskillmean, trueskillstandarddeviation from player order by id;
  trueskillmean   | trueskillstandarddeviation 
------------------+----------------------------
 23.8237282891981 |           6.76354490103157
 28.9495140233292 |           6.76354490103157
 23.6133788437364 |           7.18902494409944
 26.1762717108019 |           6.76354490103157
 21.5563971787002 |           7.66297758406055
(5 rows)

postgres=#  select * from team order by player2_id, player1_id;
  trueskillmean   | trueskillstandarddeviation | player2_id | player1_id 
------------------+----------------------------+------------+------------
  26.528121372667 |           6.07906745026758 | U12GF1WL9  | U0U7JNQH2
 29.2056413921206 |           7.19471815777177 | UGKMPGZUY  | U12GF1WL9
  23.471878627333 |           6.07906745026758 | UGKMPGZUY  | U12SQMB5W
 20.7943586078794 |           7.19471815777177 | UGZT9UB3L  | U0U7JNQH2
(4 rows)
```
*after calling calcTeamSkills via slack:*
```

postgres=# select teamaplayer1skillchange, teamaplayer2skillchange, teamaskillchange, teambplayer1skillchange, teambplayer2skillchange, teambskillchange  from tournament order by date;
 teamaplayer1skillchange | teamaplayer2skillchange | teamaskillchange | teambplayer1skillchange | teambplayer2skillchange |  teambskillchange  
-------------------------+-------------------------+------------------+-------------------------+-------------------------+--------------------
        4.61855737022626 |        4.61855737022626 | 7.62148691880531 |       -1.32911772498836 |       -1.32911772498836 | -0.789795865435902
       -2.57225335878822 |       -2.57225335878822 | 5.23467627653022 |        6.14866404895366 |        6.14866404895366 |    8.2909190218643
        3.83933299626919 |        3.83933299626919 | 7.62148691880531 |       -1.28645273786195 |       -1.43253557348142 | -0.789795865435902
(3 rows)

postgres=# select trueskillmean, trueskillstandarddeviation from player order by id;
  trueskillmean   | trueskillstandarddeviation 
------------------+----------------------------
 23.8237282891981 |           6.76354490103157
 28.9495140233292 |           6.76354490103157
 23.6133788437364 |           7.18902494409944
 26.1762717108019 |           6.76354490103157
 21.5563971787002 |           7.66297758406055
(5 rows)

postgres=# select * from team order by player2_id, player1_id;
  trueskillmean   | trueskillstandarddeviation | player2_id | player1_id 
------------------+----------------------------+------------+------------
  26.528121372667 |           6.07906745026758 | U12GF1WL9  | U0U7JNQH2
 29.2056413921206 |           7.19471815777177 | UGKMPGZUY  | U12GF1WL9
  23.471878627333 |           6.07906745026758 | UGKMPGZUY  | U12SQMB5W
 20.7943586078794 |           7.19471815777177 | UGZT9UB3L  | U0U7JNQH2
(4 rows)
```

File: frontend/src/pages/teamStats/teamStats.html [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+            <ion-col>
+              <ion-avatar item-left>
+                <img src="{{ teamStat?.team.player2.avatarImage }}">
+              </ion-avatar>
+            </ion-col>
+            <ion-col>
+              {{ teamStat?.team.player2.name }}
+            </ion-col>
+            <ion-col>
+              {{ roundSkill(teamStat?.team.trueSkillMean) }}
```
Comment (bratke) [inline frontend/src/pages/teamStats/teamStats.html]: try something like

team.trueSkillMean - 3 * team.trueSkillStandardDeviation

Comment (bratke) [reply frontend/src/pages/teamStats/teamStats.html]: .toFixed(2)

**Review by bratke — COMMENTED** [review]
-

**Comment by bratke** [comment]
this implements #76

**Pull request merged**